### PR TITLE
Commits 101

### PR DESCRIPTION
**Pull request by 0xInd3x**
this pull request adds ALEXZ4

### CODE REVIEW

File: _posts/0000-01-02-ALEXZ4.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (ALEXZ4) [inline _posts/0000-01-02-ALEXZ4.md:5]: Hi file change

**Review by 0xInd3x — COMMENTED** [review]
changed

**Review by 0xInd3x — COMMENTED** [review]
add text

**Review by 0xInd3x — COMMENTED** [review]
changed this text

File: _posts/0000-01-02-ALEXZ4.md (hunk 1 above)
Comment (ALEXZ4) [inline _posts/0000-01-02-ALEXZ4.md:5]: new

**Review by 0xInd3x — COMMENTED** [review]
changed it

**Review by 0xInd3x — COMMENTED** [review]
change this

**Review by 0xInd3x — COMMENTED** [review]
please change this line for you
what the issues ?

**Review by 0xInd3x — COMMENTED** [review]
ALEXZ4 change it

**Pull request closed without merging**